### ISSUE TITLE
Terminal fonts are "fuzzy"

### ISSUE THREAD

**Issue by vvs** [Microsoft/vscode#34362]
- VSCode Version: Code - Insiders 1.17.0-insider (38853832f83c75a76412d3586dfb8bdeb97a8777, 2017-09-14T07:58:39.362Z)
- OS Version: Windows_NT x64 10.0.15063
- Extensions: 

Extension|Author (truncated)|Version
---|---|---
Bookmarks|ale|0.16.0
project-manager|ale|0.21.1
vscode-eslint|dba|1.3.1
gitlens|eam|5.0.0
EditorConfig|Edi|0.10.1
prettier-vscode|esb|0.23.1
beautify|Hoo|1.1.1
vscode-stylefmt|mrm|2.5.0
atom-keybindings|ms-|3.0.1
vetur|oct|0.9.7
stylelint|shi|0.29.0
vscode-todo-highlight|way|0.5.11
vscode-jumpy|wma|0.3.1
---
Basically, the main editor uses subpixel anti-aliasing, making the font look good.
The terminal uses grayscale AA, making the font look fuzzy. I am using Fira Code Light.

Also, I didn't notice the fuzziness of the terminal fonts in the past, but in today's Insiders build it is very noticeable.

Here's the screenshot. It doesn't look *that* bad as compared to what I actually see on my screen :)

![vs-code-font-rendering](https://user-images.githubusercontent.com/9781/30422187-77c394b4-993f-11e7-81aa-148d836655dc.png)

**Comment by vvs** [comment]
Ah, looks like duplicate of #17827.

But my case is on Windows, not Linux.
